CONTRIBUTING.md

```
@@ -0,0 +1,31 @@
+# Contributing
+
+See [docs/ARCHITECTURE.md](docs/ARCHITECTURE.md) for how the codebase is laid out,
+including a walkthrough of adding a second task tracker or calendar backend.
+
+## Setup
+
+```bash
+$ uv sync --locked
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)